app/code/Magento/Integration/Model/ResourceModel/Oauth/Token/RequestLog.php

```diff
@@ -58,11 +58,21 @@ protected function _construct()
      */
     public function getFailuresCount($userName, $userType)
     {
+        $date = (new \DateTime())->setTimestamp($this->dateTime->gmtTimestamp());
+        $dateTime = $date->format(\Magento\Framework\Stdlib\DateTime::DATETIME_PHP_FORMAT);
+        
         $select = $this->getConnection()->select();
         $select->from($this->getMainTable(), 'failures_count')
-            ->where('user_name = :user_name AND user_type = :user_type');
+            ->where('user_name = :user_name AND user_type = :user_type AND lock_expires_at > :expiration_time');
 
-        return (int)$this->getConnection()->fetchOne($select, ['user_name' => $userName, 'user_type' => $userType]);
+        return (int)$this->getConnection()->fetchOne(
+             $select, 
```

Comment:
It looks like the indenting was a bit out. Reduce indenting by one space on lines 69 - 74

Reply:
Also line 69, 70, 71 & 72 have a whitespace at the end of the line. These need to be removed.
I would suggest setting up PHPSniffer locally so you can check BEFORE committing. If you rely on the testing here then it could take a long time.

If you can get these cleaned up and the inherit one still exists we can ask one oft he Magento guys why you are getting a fail on code you didn't touch!

Reply:
Thanks @BarnyShergold , I am trying to install PHPsniffer locally and check why those tests are failing. I will let you know what i found out.